src/seedu/addressbook/common/Messages.java

```diff
@@ -6,6 +6,7 @@
 public class Messages {
 
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MESSAGE_PRICE_LIMIT_EXCEEDED = "Price cannot be so large! ";
```

Comment:
Incorrect message
First of all, we don't know if the input is even a number (the user can just input "asdfghjklsdfghjkl") or if it is indeed too large or not (the user type "$000000000000000001.00")
Should be changed to "The price input has cannot have more than <PRICE_STRING_MAX_LENGTH> digits/symbols/characters"